### DISCUSSION TITLE
What is the unit in `colour.SDS_ILLUMINANTS`? How to use my own measured spectrum?

### DISCUSSION THREAD

**Discussion by kikdu** [colour-science/colour#1311] [Q&A]
Hello,

I want to perform a spectral upsampling using

`colour.XYZ_to_sd(XYZ, method="Jakob 2019", cmfs=cmfs, illuminant=illuminant)`

Instead of using one of the illuminants in `colour.SDS_ILLUMINANTS`, I want to use the spectrum of a white reference that I measured with a spectroradiometer (which is the estimation of the illuminant in my natural scene). 

What is the unit in colour.SDS_ILLUMINANTS and how to transform my data to match the expected data ?

Thank you !

**Comment by thomasmansencal** [comment]
Hi @kikdu,

The illuminants typically represent irradiance. Reflectance is always normalised and given as a percentage between [0, 100] and independent of the measurement illuminant.

Cheers,

Thomas
